### ISSUE TITLE
Unable to test podman on mac with provider applehv due to vfkit missing

### ISSUE THREAD

**Issue by adrianriobo** [containers/podman#21064]
### Issue Description

Trying to start a podman machine on macos with applehv it ends up on error due to missing vfkit binary

### Steps to reproduce the issue

Steps to reproduce the issue
1. export CONTAINERS_MACHINE_PROVIDER= applehv
2. podman machine init
3. podman machine start 

### Describe the results you received

Got an error:

```bash
Error: could not find "vfkit" in one of {[/opt/podman/qemu/bin $BINDIR/../libexec/podman /usr/local/opt/podman/libexec/podman /opt/homebrew/opt/podman/libexec/podman /opt/homebrew/bin /usr/local/bin /usr/local/libexec/podman /usr/local/lib/podman /usr/libexec/podman /usr/lib/podman] {<nil>}}.  To resolve this error, set the helper_binaries_dir key in the `[engine]` section of containers.conf to the directory containing your helper binaries.
```

### Describe the results you expected

Podman machine running successfully with applehv provider

### podman info output

```yaml
podman v4.8.2 installed from pkg
```

### Podman in a container

No

### Privileged Or Rootless

Rootless

### Upstream Latest Release

Yes

### Additional environment details

Additional environment details

### Additional information

Additional information like issue happens only occasionally or issue happens with a particular architecture or on a particular setting

**Comment by benoitf** [comment]
hello, for now the podman installer is not installing vfkit

you need to install it on your own using https://github.com/crc-org/vfkit/releases

**Comment by rhatdan** [comment]
@gbraad @baude @ashley-cui PTAL

**Comment by gbraad** [comment]
@rhatdan I also noticed this; but it is more a question if when this becomes the default. Adding this to the installer is an easy step.

**Comment by rhatdan** [comment]
I think the goal is podman 5.0

**Comment by jfrantzius** [comment]
@rhatdan following your advice from https://github.com/containers/podman/issues/20776#issuecomment-1867633199 , I erased anything podman-related from brew (because I also had the frozen applehv machine startup), and used the Podman Desktop .dmg installer.

Does your comment above mean applehv without manually installing vfkit will work only with Podman 5.0 upwards?

**Comment by rhatdan** [comment]
Yes, that is my understanding.

**Comment by jfrantzius** [comment]
I'm worrying a bit when applehv will be easily consumable for our developers, as 5.0 sounds a bit far off... Is there perhaps some release planning happening in the open that you are aware of? Maybe 5.0 will then have applehv as default?

On the other hand, I hope brew will still be supported (maybe less appropriate for early testing). Is vfkit then a brew dependency of Podman? I'm asking because I got applehv to start and say hello via brew (before it broke down)

**Comment by rhatdan** [comment]
podman 5.0 is scheduled for February.  There are fixes comming soon for the QEMU issues, I have been told.

**Comment by gbraad** [comment]
Right, a fix for qemu is on it's way. But for vfkit, we are currently looking at how to handle the inclusion of the vfkit driver binary. @cfergeau and @baude might have more information on how to handle this.

**Comment by ashley-cui** [comment]
I can take a look at including it in the pkginstaller

**Comment by jfrantzius** [comment]
Hi @ashley-cui , is vfkit per chance now included in the 4.9 .pkg installer?

**Comment by ashley-cui** [comment]
@jfrantzius  Haven't gotten to it yet, I'll try to get to it asap. In the meantime, VKFit can be installed via

```
brew tap cfergeau/crc
brew install vfkit
```

**Comment by Luap99** [comment]
This is fixed with the 5.0 installers I assume
